libs/community/langchain_community/vectorstores/meilisearch.py

```diff
@@ -67,6 +67,7 @@ class Meilisearch(VectorStore):
             embeddings = OpenAIEmbeddings()
             vectorstore = Meilisearch(
                 embedding=embeddings,
+                embedders=embedders,
```

Comment:
what are embedders? could we show how you'd define embedders in this example?

Reply:
In Meilisearch versions greater than 1.6, to use vector search, you must first configure the embedders index setting. This setting is a dictionary structure in which each key corresponds to an embedder. 

The configuration options within an embedder object vary depending on the value assigned to its source field, which specifies the third-party service used for generating embeddings. In this context, the source is `userProvided`.

For `userProvided` sources, it's required to include a dimensions field, which outlines the size of the vectors.

An example configuration for embedders could look like this:

```
embedders = {
    "theEmbedderName": {
        "source": "userProvided",
        "dimensions": "1536"
    }
}
```

The use can define multiple embedders per index to accommodate more demanding use cases. For example, a user can define one embedder to generate text from a document image, and another embedder to translate the original document text fields to let users speaking other languages search products in their native language.

Reply:
gotcha! should we that snippet in the docstring example?